docs/configs/janusgraph-cfg.md

```diff
@@ -514,7 +514,6 @@ HBase storage options
 
 | Name | Description | Datatype | Default Value | Mutability |
 | ---- | ---- | ---- | ---- | ---- |
-| storage.hbase.compat-class | The package and class name of the HBaseCompat implementation. HBaseCompat masks version-specific HBase API differences. When this option is unset, JanusGraph calls HBase's VersionInfo.getVersion() and loads the matching compat class at runtime.  Setting this option forces JanusGraph to instead reflectively load and instantiate the specified class. | String | (no default value) | MASKABLE |
```

Comment:
I guess we just need to double check that we won't need this configuration when we bring support for HBase 3

Reply:
I don't think as they say client should compatible: https://github.com/apache/hbase/blob/master/src/main/asciidoc/_chapters/upgrading.adoc#hbase.rolling.upgrade